tutorials/interoperability/remote-batched-evm-calls.md

```diff
@@ -0,0 +1,230 @@
+
+![Sending the XCM message from the Relay Chain to Moonbase Alpha for the batched EVM call](/images/tutorials/interoperability/remote-batched-evm-calls/remote-batched-evm-calls-6.png)
+
+And that is it! You've sent an XCM message, which performed a remote EVM call to the batch precompile via XCM and resulted in mints of `Mars` and `Neptune` ERC-20 tokens. But let's go into more detail about what happened.
+
+This action will emit different events. The first one is the only relevant [in the relay chain](https://polkadot.js.org/apps/?rpc=wss://frag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/explorer/query/0x85cad5f3cef5d578f6acc60c721ece14842be332fa333c9b9eafdfe078bc0290){target=_blank}, and it is named `xcmPallet.Sent`, which is from the `xcmPallet.send` extrinsic. In [Moonbase Alpha](https://polkadot.js.org/apps/?rpc=wss://wss.api.moonbase.moonbeam.network#/explorer/query/0x1f60aeb1f2acbc2cf6e19b7ad969661f21f4847f7b40457c459e7d39f6bc0779){target=_blank}, the following events emitted by the `parachainSystem.setValidationData` extrinsic (where all the inbound XCM messages are processed) are of interest:
+
+ - `parachainSystem.DownwardMessagesReceived` — states that there was an XCM message received
+ - `evm.Log` — internal events emitted by the different contract calls. The structure is the same: contract address, the topics, and relevant data
+ - `ethereum.Executed` — contains information on the `from` address, the `to` address, and the transaction hash of an EVM call done
+ - `polkadotXcm.AssetsTrapped` — flags that some assets were in holding and were not deposited to a given address. If the `Transact` XCM instruction does not exhaust the tokens allocated to it, it will execute a [`RefundSurplus`](https://github.com/paritytech/xcm-format#refundsurplus){target=_blank} after the XCM is processed. This instruction will take any leftover tokens from the execution bought and put them in holding. We could prevent this by adjusting the fee provided to the `Transact` instruction, or by adding the instruction right after the `Transact`
+ - `dmpQueue.ExecutedDownward` — states the result of executing a message received from the relay chain (a DMP message). In this case, the `outcome` is marked as `Complete`
+
+Our XCM was successfully executed! If you visit [Moonbase Alpha Moonscan](https://moonbase.moonscan.io/){target=_blank} and search for [the transaction hash](https://moonbase.moonscan.io/tx/0x797f9a75257cf1b6aa0cf7a3b59758402b6bb603a27de86450e2177877aaa889){target=_blank}, you'll find the call to the Batch precompile that was executed via the XCM message. Note that you can only call the `mint` commands once per hour per planet. If you wish to experiment further and make additional mint calls, simply change the destination contract address to a different planet when configuring the batched call.
```

Comment:
ah ok just seeing now that this shows the tx went through, perhaps add this up above where i made the other comment about how users can check that everything works as expected